### PR TITLE
Please check it out

### PR DESCRIPTION
**Pull request by Amit833**

### CODE REVIEW

File: amit/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// 1
```
Comment (ciamac-da) [inline amit/index.js:1]: Here is the solutions!
// 1
// NaN stands for Not-a-Number.
console.log(isNaN("hello"));
console.log(isNaN(3));
console.log(typeof NaN);
// Type of NaN is a number

// 2
console.log(Infinity / 0);
console.log(Infinity / Infinity);
console.log(1 / 0);

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Please add the solutions that I sent you below!

File: amit/index.js (hunk 1 above)
Comment (Amit833) [reply amit/index.js:1]: console.log(typeof NaN);
// Type of NaN is a number
I don'n understand these lines

Comment (ciamac-da) [reply amit/index.js:1]: Well NaN(Not a Number) is a property that checks! This property indicates that a value is not a legal number!
So using typeof we can find out which type of data we have. Data Type of NaN is actually Number!:)
https://javascriptrefined.io/nan-and-typeof-36cd6e2a4e43

**Review by ciamac-da — APPROVED** [review]
Well NaN(Not a Number) is a property that checks! This property indicates that a value is not a legal number!
So using typeof we can find out which type of data we have. Data Type of NaN is actually Number!:)
https://javascriptrefined.io/nan-and-typeof-36cd6e2a4e43

**Pull request merged**